seaworthy/tests-core/test_logs.py

```diff
@@ -107,3 +110,86 @@ def test_str(self):
             str(matcher),
             "SequentialLinesMatcher(matched=[EqualsMatcher('foo'), "
             "RegexMatcher('^bar')], unmatched=[])")
+
+
+class FakeLogsContainer:
+    """
+    A container object stub that emits canned logs.
+    """
+
+    def __init__(self, log_entries, expected_kw=None):
```

Comment:
I don't like mutable defaults, because they're shared between all calls. I should probably use `self._expected_kw`.